runtime/jcl/cl_se9/module.xml

```diff
@@ -103,7 +103,7 @@
 			<makefilestub data="UMA_ENABLE_ALL_WARNINGS=1"/>
 			<makefilestub data="UMA_TREAT_WARNINGS_AS_ERRORS=1"/>
 			<makefilestub data="ifneq ($(OPENJ9_BUILD),true)"/>
-			<makefilestub data="jclcinit$(UMA_DOT_O) : CFLAGS += -DOPENJ9_REPO='&quot;J9VM     - &quot;'"/>
+			<makefilestub data="jclcinit$(UMA_DOT_O) : CFLAGS += -DVENDOR_VERSION=1 -DOPENJ9_REPO='&quot;J9VM     - &quot;'"/>
```

Comment:
I think VENDOR_VERSION should only be defined when its not a OPENJ9_BUILD.
-DVENDOR_VERSION should be sufficient.